webui/module/Application/language/de_DE.po

```diff
@@ -1685,7 +1685,7 @@ msgstr "Bitte wählen Sie einen Client"
 #: ../module/Restore/src/Restore/Form/RestoreForm.php:226
 #: ../module/Restore/src/Restore/Form/RestoreForm.php:242
 msgid "Please choose a restore job"
-msgstr "Bitte wählen Sie einen Rücksicherungsauftrag"
+msgstr "Bitte wählen Sie einen Rücksicherungjob"
```

Comment:
Either this or https://github.com/bareos/bareos/blob/cacbac739481110bfc9d8ed01271deb7d1585aff/webui/module/Application/language/de_DE.po#L1076 is not correct.  Im not sure which but the binnen-s makes it sound more natural.

Reply:
Good catch, corrected the typo.